test/utility/util_random.c

```diff
@@ -27,6 +27,24 @@
  */
 #include "test_util.h"
 
+/*
+ * testutil_random --
+ *     Return a random number, without the need to specify a randomness state (i.e., it can be
+ *     NULL). If the randomness state is not provided, generate a completely random number.
+ */
+uint32_t
+testutil_random(WT_RAND_STATE *rnd)
+{
+    WT_RAND_STATE rnd_inner;
+
+    if (rnd != NULL) {
+        return __wt_random(rnd);
+    } else {
+        __wt_random_init_seed(NULL, &rnd_inner); /* The session argument is not really used. */
```

Comment:
(very) minor nit: The session argument _is_ used, if not NULL, to ensure session's have monotonically increasing clocks. So maybe say that the session arg "is not required" or that the function "accepts a NULL session"?